docs/source/administrator/upgrading.md

```diff
@@ -100,19 +76,27 @@ will be performed automatically when you do a `helm upgrade`.
 4. Do a [`helm upgrade`](helm-upgrade-command). This should perform the database upgrade needed.
```

Comment:
TODO: test the above MySQL/PostgreSQL still works

Reply:
@yuvipanda do you happen to know if this has worked for you when updating to a jupyterhub 2 versioned z2jh with a postgresql server?

I can test the upgrade part pre-release, for now, assume it works!

Reply:
I used SQLite everywhere, no postgres. I told believe it should just work.